src/scss/App.scss

```diff
@@ -0,0 +1,76 @@
+@import '_variables.scss';
+@import './libs/normalize';
+@import './components/all';
+
+
+.app{
+  display: flex;
+  &__footer{
+    display: flex;
+    flex-direction: row;
+  position: absolute;
+    bottom: 1%;
+    right: 85%;
+
+    span{
+      letter-spacing: 0.2em;
+      font-size: 15px;
+      color:#F1CDB3
```

Comment:
if you started to use color variables like $white - you should use them everywhere